### PR TITLE
Timeline Modal

### PR DESCRIPTION
**Pull request by phuang00**
This PR:

- creates the `DocumentModal` and children component to display an article of the timeline
- updates the project to use Bootstrap 5 and react-bootstrap 2.1.1 and updates previous navbar stylings
- creates a sample timeline page to show how `DocumentModal` is used (found at http://localhost:8000/timeline-test)

### CODE REVIEW

File: frontend/components/DocumentModal.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    return (
+        <Modal show={modalShow} onHide={toggleHide} fullscreen={true}>
+            <Modal.Header className="text-center" closeButton>
+                <Modal.Title className="w-100">
+                    <h1>{document ? document.title : ""}</h1>
+                </Modal.Title>
+            </Modal.Header>
+            <Modal.Body>
+                {document === null
+                    ? <div>Loading document...</div>
```
Comment (anastasia) [inline frontend/components/DocumentModal.js:75]: great use of the ternary operator!

**Review by anastasia — CHANGES_REQUESTED** [review]
Hey, really great work. 
I only have one `to do` but it's a big one, and it is also my fault — for which I apologize. I will post this in the group channel also.
For this project, we have decided to move away from importing STYLES directly into our React components. The main reason behind that is that is it's really difficult to use in development — debugging styles and scoping become very complex when our app is compiled and we're looking at something that's called `zgnKfzkW1Yy6JWrJlX1mHA=="` instead of `class-1`. 
If you could please move to using regular old scss, that would be great! We can have all the document styles live in `/scss/document.scss` and have that stylesheet be imported in `scss/index.scss`. This does mean that the styles become available to the entire app, so we need to be careful with scoping the styles. 
Happy to pair on this, I know this is quite a big task. Let me know if you have any questions or if anything needs clarification.

**Comment by phuang00** [comment]
We've updated the styles for the timeline modal. They're all under `/scss/document.scss` now!

**Comment by anastasia** [comment]
Great, thank you!

**Pull request merged**